src/gui/ui_settings.c

```diff
@@ -381,14 +381,15 @@ enum {
   SETTINGS_ENABLE_STREAM_OPTIMIZE,
   SETTINGS_SAVE_DEBUG_LOG,
   SETTINGS_DISABLE_POWERSAVE,
+  SETTINGS_DISABLE_VSYNC,
   SETTINGS_ENABLE_MAPPING,
   SETTINGS_BACK_DEADZONE,
   SETTINGS_SPECIAL_KEYS,
   SETTINGS_MOUSE_ACCEL,
 };
 
-
-enum {
+enum
+{
```

Comment:
```suggestion
enum {
```
return original :)

Reply:
What do you mean?

Reply:
you changed
```
enum {
```
to
```
enum
{
```

in this project, no one used that formatting. so you need to restore it.

Reply:
Ah, didn't check. It was Visual Studio Code auto-formatting. Fixing.